app/controllers/menu_items_controller.rb

```diff
@@ -0,0 +1,48 @@
+class MenuItemsController < ApplicationController
```

Comment:
I know that I don't do this, but can we add a MenuItemsControllerSpec so that we can test that the required params are actually required, and that the GET returns all the menu_items, POST creates one, etc.

If you look at request/sellers_spec.rb, I've been using the requests spec as an integration test and it was a bad idea because we end up with a super test file. It would be better to unit test each controller